api/blueprints/subtunes_api.py

```diff
@@ -49,20 +33,20 @@ def get_image_url(img_path):
 #         return False, e
 
 # delete subtune image from aws s3 bucket
-def delete_file_from_s3(file_url):
+# def delete_file_from_s3(file_url):
```

Comment:
how are we deleting images in s3? if we do it somewhere else we should delete this commented code